### ISSUE TITLE
Reads preprocess

### ISSUE THREAD

**Issue by guandailu** [google/deepvariant#569]
**Have you checked the FAQ? https://github.com/google/deepvariant/blob/r1.4/docs/FAQ.md**:

**Describe the issue:**
 I am moving GATK to DeepVariant, but I do wondering if we need to do same data preprocessing as GATK, including deduplicate and Variant Recalibration (especially this step)? The later needs run two steps using functions, including VariantRecalibrator, ApplyVQSR.?

**Comment by AndrewCarroll** [comment]
Hi @guandailu 

We have investigated the use of the MarkDuplicates step. In WGS and exome samples we looked at, we observe no measurable difference at coverages 30x and higher. For coverages lower than 30x, we observe a very slight advantage for MarkDuplicates. For this reason in our [best practices recommendation](https://github.com/google/deepvariant/blob/r1.4/docs/trio-merge-case-study.md), we indicate duplicate marking as optional. 

In terms of other preprocessing steps, we observe that running Base Quality Score Recalibrator (BQSR) consistently results in a slight reduction of accuracy (likely because DeepVariant has already learned how to use the underlying concept to recalibrate qualities as it considers calling).

We have a strong recommendation NOT to run Variant Score Quality Recalibration (VQSR). We observe large reductions in accuracy, particularly in recall of rare variants. If you would like to filter for higher precision than the default calls, we recommend using the Genotype Quality (GQ) field, setting a higher threshold based on your desire for specificity.
